context/app/static/js/components/Routes.jsx

```diff
@@ -65,6 +66,14 @@ function Routes(props) {
       </Container>
     );
   }
+
+  if ('markdown' in flaskData) {
```

Comment:
Actually should we also check the url path or is that difficult given the variety of paths for the documentation?